buffer-api/src/main/java/io/netty/buffer/api/MemoryManagers.java

```diff
@@ -59,6 +62,25 @@ static MemoryManagers getManagers() {
         return loader.stream();
```

Comment:
It may not be expected that a new `MemoryManagers` is created each time this method is called - perhaps add some documentation mentioning this?